data/frameSimulator/src/MercuryFrameSimulatorPlugin.cpp

```diff
@@ -0,0 +1,199 @@
+#include <string>
+
+#include "MercuryFrameSimulatorPlugin.h"
+#include "MercuryDefinitions.h"
+
+#include <cstdlib>
+#include <time.h>
+#include <iostream>
+#include <algorithm>
+#include <boost/lexical_cast.hpp>
+
+#include "version.h"
+
+namespace FrameSimulator {
+
+
+    /** Construct an MercuryFrameSimulatorPlugin
```

Comment:
Feel free to ignore this one as it has nothing to do with the code but "an Mercury.." should be "a Mercury..".

Is there a grammar Nazi emoticon? :-p

Reply:
I don't think pointing out a typo qualifies you as a grammar Nazi 🤷‍♂️ 👍  

Guess this is a hangover from EXCALIBUR, HEXITEC, ....